build.gradle

```diff
@@ -1,16 +1,50 @@
-apply plugin: 'java'
-apply plugin: 'jacoco'
+buildscript {
+    repositories {
+        jcenter()
+    }
 
-jacoco {
-    version = "0.7.1.201405082137"
+    dependencies {
+        classpath 'com.android.tools.build:gradle:1.2.3'
+        classpath 'org.robolectric:robolectric-gradle-plugin:1.1.0'
+        classpath 'com.jakewharton.sdkmanager:gradle-plugin:0.12.+'
+    }
 }
 
-sourceCompatibility = 1.6
-targetCompatibility = 1.6
-version = '0.1.0'
+apply plugin: 'android-sdk-manager'
```

Comment:
It is not used directly, but it will be triggered if you dont have an appropriate sdk version installed.
Please read here: https://github.com/JakeWharton/sdk-manager-plugin
Sounds like really useful stuff :)